lib/packer/env.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Packer
+  class Env
+    DEFAULT = "development"
+
+    delegate :config_path, :logger, to: :@packer
+
+    def self.inquire(packer)
+      new(packer).inquire
+    end
+
+    def initialize(packer)
+      @packer = packer
+    end
+
+    def inquire
+      fallback_env_warning unless current
```

Comment:
I'm not sure I agree with this. Why would we ever want to continue if the environment is unknown?

Reply:
Given that we are supporting only two applications Sinatra and Rails, we have a very explicit way of fetching the ENV from them. This was here for convenience when developing. I guess we can add a `PACKER_ENV` for this purpose rather then going to a default?